.rubocop.yml

```diff
@@ -39,6 +39,9 @@ Style/AccessModifierDeclarations:
 Style/FrozenStringLiteralComment:
   EnforcedStyle: always
 
+Style/NumericPredicate:
+  Enabled: false
```

Comment:
I killed this one biggest IMO it makes code less readable and is bad.

It basically requires that something like `if num_retries > 0` become `if num_retries.positive?`.

Full description here: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/NumericPredicate